rep-0149.rst

```diff
@@ -619,6 +619,8 @@ Attributes
   * All literals are also treated as strings.
   * The resulting expression is evaluated as a Python interpreter would
     evaluate it.
+    Please not that the comparison operators only do a *string* comparison and
```

Comment:
Typo: ``not`` should be ``note``

Reply:
Thanks. Fixed in 8bbfda1cfc8c621d283217e771e2d88167b2668a.